project-base/storefront/components/Blocks/StoreList/StoreListItem.tsx

```diff
@@ -48,12 +50,21 @@ export const StoreListItem: FC<StoreListItemProps> = ({ store, isSelected }) =>
         >
             <div className="flex items-center justify-between gap-3.5">
                 <div className="w-full items-center justify-between xl:flex">
-                    <div className="max-xl:mb-2.5">
+                    <div className="max-xl:mb-2.5 xl:w-[215px]">
                         <h5>{store.name}</h5>
                         <p className="mt-1.5 text-xs">
                             {store.street}, {store.postcode} {store.city}
                         </p>
                     </div>
+                    {store.distance && (
+                        <div className="max-xl:hidden">
```

Comment:
similar thing here. I think you can put the class name `max-xl:hidden` in-between `p` classes